### DISCUSSION TITLE
When is the current status of v14?

### DISCUSSION THREAD

**Discussion by kmcclellan** [ChilliCream/hotchocolate#6800] [General]
I've noticed v14 has had prerelease versions in the works for a few months now. Any estimate on when this will be released or update on what the current blockers are?

**Comment by kmcclellan** [comment]
Hello @michaelstaib !

**Reply by tobias-tengler** [reply]
It's in the works. Is there any feature coming you care about or why is the timeline of v14 important to you?

**Reply by kmcclellan** [reply]
Project planning. I'd like to reduce the amount of new code I write targeting v13 if there are about to be breaking changes.

**Reply by bbarry** [reply]
I was wondering if there is a known list of breaking / planned breaking changes. In a quick upgrade test from 13.9.0 to 14.0.0-p.87 I had only 1 compile time breaking change:

* `IIdSerializer` interface was removed and a new interface was added that was a little different

But roughly half my tests fail because the service collection scope for data loaders has changed (in 13.9.0 a dependency registered as a scoped service received the same instance in a resolver method as it gets in a source generated data loader, but in 14 the data loader resolves a new instance, presumably because it is in a different scope somehow). In other words in 13, `services.AddScoped<T>()` registers something that resolves one instance per http request execution but in 14 this is not true.

**Reply by smyrick** [reply]
Is there a plan for maybe and RC or stable pre-release? We have many users looking forward to this fix in a stable release: https://github.com/ChilliCream/graphql-platform/pull/6838

**Reply by ghost** [reply]
I am looking forward to this pr in version v14 #7062 currently cannot send any queries because the id parameter is always present on my graphql querries
